src/Copy/Text.elm

```diff
@@ -313,53 +316,53 @@ t key =
             "[cCc] Help in times of COVID"
 
         HelpSelfCategory1Resource1Title ->
-            "[cCc] Economic abuse and the coronavirus outbreak"
```

Comment:
I think these need to be the examples I put in to help Stuart and Georgia see in context.